test/extended/testdata/builds/test-build-search-registries.yaml

```diff
@@ -0,0 +1,15 @@
+apiVersion: build.openshift.io/v1
+kind: BuildConfig
+metadata:
+  name: ubi
+spec:
+  source:
+    # ubi8/ubi:latest only resolves to an image on registry.redhat.io or registry.access.redhat.com.
+    dockerfile: FROM ubi8/ubi:latest
```

Comment:
I think we need to add this to the list of allowed images. This may require some level of scrutiny because this test covers functionality that impacts disconnected use cases:

https://github.com/openshift/origin/blob/master/test/extended/util/image/image.go

Reply:
I picked ubi8/ubi because it was already being used elsewhere:  https://github.com/openshift/origin/blob/master/test/extended/testdata/builds/test-build-proxy.yaml#L101

you in fact introduced that with https://github.com/openshift/origin/commit/bb063138df44488b9fb0f02040fee87a1a63cd95

And that PR https://github.com/openshift/origin/pull/25778 merged in April, at which point https://github.com/openshift/origin/blob/master/test/extended/util/image/image.go was in full throttle, reviewing its git history

So either we overlooked the mirroring hoopla back then, or ubi8/ubi is so core/essential it gets pulled in elsewhere.

But if it was an oversight, I'm more inclined to use the busybox entry already in there, k8s.gcr.io/e2e-test-images/busybox:1.29-1

Reply:
the busybox image breaks the premise of the test. `k8s.gcr.io` is not a container registry on our search list, and moreover `busybox` is a default image on Docker Hub.

Reply:
> So either we overlooked the mirroring hoopla back then, or ubi8/ubi is so core/essential it gets pulled in elsewhere.

It's possible the ubi layers are present in other images we use. That would depend on a lot of things.
Feels like there was an oversight here, because we indirectly reference the image in our test. This is also explicitly a part of the build suite, which means it isn't run in the normal conformance test.

I think we are OK merging here, then having a follow-up PR that adds the image to the mirror process.

Reply:
> the busybox image breaks the premise of the test. `k8s.gcr.io` is not a container registry on our search list, and moreover `busybox` is a default image on Docker Hub.

duh - wasn't thinking :-/ thanks @adambkaplan

Reply:
> > So either we overlooked the mirroring hoopla back then, or ubi8/ubi is so core/essential it gets pulled in elsewhere.
> 
> It's possible the ubi layers are present in other images we use. That would depend on a lot of things. Feels like there was an oversight here, because we indirectly reference the image in our test. This is also explicitly a part of the build suite, which means it isn't run in the normal conformance test.
> 
> I think we are OK merging here, then having a follow-up PR that adds the image to the mirror process.

sounds like a plan thanks